events/how-to-start-a-class.md

```diff
@@ -43,24 +43,3 @@ If you are are interested in running a course on our premises then contact [even
 
 # Want to start something?
```

Comment:
With the deletion, there's no content after this header, which isn't good...

Reply:
I'll fix it and commit again, sorry!

Reply:
Done, let me know if it's ok

Reply:
It's showing in the log that it's been committed to this branch but it's not listed in commit. Not sure what to do now :p

Reply:
Nevermind, got it!